torch/lib/c10d/test/ProcessGroupGlooTest.cpp

```diff
@@ -581,6 +581,20 @@ void testRecv(const std::string& path) {
   EXPECT_TRUE(recvCompleted);
 }
 
+void testSetGet(const std::string& path) {
+  const auto size = 2;
+  auto tests = CollectiveTest::initialize(path, size);
+
+  constexpr uint64_t tag = 0x1337;
+  // test that get() gets the same value as the one that was set()
+  auto selfRank = 0;
+  std::vector<uint8_t> testVector = testVector(10, 1);
+  auto& store = tests[selfRank].getProcessGroup().store_;
```

Comment:
Right, this won't work. I think it's fine to expose a simple `store()` accessor as I don't really see a need for the store to be private.